openreview/conference/templates/bidWebfield.js

```diff
@@ -10,8 +10,6 @@ var SUBMISSION_ID = '';
 var BID_ID = '';
 var SUBJECT_AREAS = '';
 
-var USER_SCORES_INVITATION_ID = CONFERENCE_ID + '/-/User_Scores';
```

Comment:
We can remove this for now but we need to remember that we may need to sort the papers by a type of edge.